src/app/login/login-modal/login-modal.component.html

```diff
@@ -0,0 +1,36 @@
+<div class="container"></div>
+<div class="col-lg-6 col-md-6 col-sm-6 col-xs-6 offset-3 float-md-center">
+  <div class="jumbotron" id="login-modal">
+    <form action="action_page.php" method="post">
```

Comment:
We should not have any php in our project since this is not a php project, I'm a little confused on where this is coming from.

Probably go look at how I designed the forms on the user-action-table component (both the .html and .ts)